pkg/crc/machine/start.go

```diff
@@ -265,8 +265,8 @@ func configureSharedDirs(vm *virtualMachine, sshRunner *crcssh.Runner) error {
 
 func (client *client) Start(ctx context.Context, startConfig types.StartConfig) (*types.StartResult, error) {
 	telemetry.SetCPUs(ctx, startConfig.CPUs)
-	telemetry.SetMemory(ctx, uint64(startConfig.Memory)*1024*1024)
-	telemetry.SetDiskSize(ctx, uint64(startConfig.DiskSize)*1024*1024*1024)
+	telemetry.SetMemory(ctx, uint64(startConfig.Memory)*1024*1024)          //#nosec G115 -- validation make sure memory value is not negative
```

Comment:
actually I haven't been able to find such >=0 checks in the  validation code?

Reply:
for memory as part of validation we use the base memory as default preset one (in case of OCP it is 10752 and for microshift 4096

```go
// ValidateMemory checks if provided Memory count is valid
func ValidateMemory(value int, preset crcpreset.Preset) error {
	if value < constants.GetDefaultMemory(preset) {
		return fmt.Errorf("requires memory in MiB >= %d", constants.GetDefaultMemory(preset))
	}
	return ValidateEnoughMemory(value)
```